packages/dom-helper/lib/main.js

```diff
@@ -580,11 +579,16 @@ function installEnvironmentSpecificMethods(domHelper) {
     // Swap in the method that doesn't do this test now that
     // we know it works.
     domHelper.protocolForURL = browserProtocolForURL;
+  } else if (typeof URL === 'object') {
+    // URL globally provided, likely from FastBoot's sandbox
+    domHelper.protocolForURL = nodeProtocolForURL;
   } else if (typeof module === 'object' && typeof module.require === 'function') {
     // Otherwise, we need to fall back to our own URL parsing.
     // Global `require` is shadowed by Ember's loader so we have to use the fully
     // qualified `module.require`.
+    /* jshint -W020 */
     URL = module.require('url');
```

Comment:
I'm happy to rework this to use a differently named var in this scope to eliminate this warning.  That should work assuming the only place we need `URL` is in [nodeProtocolForURL](https://github.com/tildeio/htmlbars/pull/439/files#diff-9c4c7dd8b7ebbbd7f60a97eaaf81dde2L614).